politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -801,6 +823,19 @@ func _main() error {
 		}
 		userDB = db
 		defer userDB.Close()
+
+	case *mysql:
```

Comment:
We also need a way to migrate the existing user database from one database type to another (ex. cockroachdb to mysql).  I'm thinking a `migrate` command would be the best way to do this.  The command would iterate through all users in database A and save each user to database B.

Reply:
👍🏽